CONTRIBUTING.md

```diff
@@ -77,13 +76,7 @@ ufmt format `git diff main --name-only`
 
 ### Type checking
 
-TorchMultimodal uses mypy for type checking. You can install mypy with the command
-
-```
-python3 -m pip install mypy
-```
-
-To perform type checking:
+TorchMultimodal uses mypy for type checking.To perform type checking:
```

Comment:
Adding a space here?
`TorchMultimodal uses mypy for type checking.(space)To perform type checking:`